test/artefacts/am-i-getting-minimum-wage/past_payment/2008-10-01/apprentice_under_19.txt

```diff
@@ -1,6 +1 @@
-
-
-There was no National Minimum Wage rate for apprentices before 1 October 2010. Your employer doesn't owe you any past payments.
-
-
-
```

Comment:
I didn't know to run the script to generate responses and expected results. I shall update the documentation. Thanks.